packages/apps-config/src/endpoints/productionRelayKusama.ts

```diff
@@ -142,6 +142,15 @@ export function createKusama (t: TFunction): EndpointOption {
           'KILT Protocol': 'wss://spiritnet.kilt.io/'
         }
       },
+      {
+        info: 'kpron',
+        homepage: 'http://apron.network/',
+        paraId: 2019,
```

Comment:
```suggestion
        isUnreachable: true,
        paraId: 2019,
```

Since there is no WASM on-chain yet for 2019, the network is technically not there yet. If uploaded soon, we can leave it without the flag.